app/src/main/java/com/github/fribourgsdp/radio/Language.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.fribourgsdp.radio
+
+enum class Language(val id: Int, val code: String, val s: String) {
+    ENGLISH(0,"en", "English"),
+    FRENCH(1,"fr", "Français");
```

Comment:
For your information, the `id` property here is useless because there exists in enums a default property called `ordinal` that already does that.

Reply:
ok it works, thanks for the information.